Readme.md

```diff
@@ -1,10 +1,64 @@
+| start-docker |<br/> -t or --trustednode <ip of node on network>, required.<br/> -f or --composefile <name of the composefile>required|<br/>- Prompts about generation of key otherwise just password if key already exists| Setups the environment variables and brings up the container |
+| stop-docker |<br/> -f or --composefile <name of the composefile>,required <br/>-v or --removevolumes, optional|| Stops the docker containers and removes volumes if one wants to clear the volumes. Externally mounted volumes won&#39;t be cleared even with -v option |
+
+## Setup/Update systemd
+
+| **Sub Commands** | **Arguments** | **Prompts** | **Comments** |
+| --- | --- | --- | --- |
+| configure-systemd ||<br/>- Prompts for radixdlt user password| Prints out instructions to edit sudoers file and add public ssh key for password less login |
+| start-systemd |<br/> -r or --release <releasetag>, required<br/> -n or --nodetype <fullnode or archivenode>, required,<br/> -t or --trustednode <ip of radixnode on network>, required.<br/> -i or --hostip <ip of the host>,required<br/> -u or --update, optional|<br/>- Displays url of node binary and nginx binary being downloaded and prompts to continue<br/>- Prompts to back up files for node service, if the below files exists<br/>-- environment<br/> -- config<br/> -- radixdlt-node.service<br/> - Prompts if user wants to setup nginx. If yes , then prompts for backup on existing nginx files<br/> - Prompts for existing nginx secrets before recreating them||
+| stop-systemd |<br/> -s or --services <nginx or radixdlt-node>, defaults=all|| Stop the service based on the option. If option not provided , it stops both |
+
+## Nginx Passwords
+
+| **Sub Commands** | **Arguments** | **Prompts** | **Comments** |
+| --- | --- | --- | --- |
+| set-admin-password |<br/> -m or --setupmode <DOCKER or SYSTEMD>, required|<br/>- Prompts for nginx admin password to be changed or to be setup||
+
+## Monitoring
+
+| **Sub Commands** | **Arguments** | **Prompts** | **Comments** |
+| --- | --- | --- | --- |
+| setup-monitoring ||| Uses hardcoded cli version to download the aritfacts. Creates external volumes for prometheus and grafana and starts up the monitoring containers |
+| stop-monitoring |<br/> -v or --removevolumes|| Stops the docker containers and removes volumes if one wants to clear the volumes. Externally created/mounted volumes won&#39;t be cleared even with -v option |
+
+### More usage instructions
+
+To list all subcommands
 ```shell script
 # To list the subcommands
 python3 nodecli.py --help
```

Comment:
shouldn't it reference the binary instead of python3 nodecli.py ? Same in other places